api/server/src/main/java/javax/websocket/server/ServerEndpoint.java

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (c) 2012, 2017 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2017 Oracle and/or its affiliates and others.
```

Comment:
Whats the "and others" stands for? Is it some Eclipse standard to mention community contributors?  Year should be changed to 2018.

Reply:
The above change (inserting ' and others') is what we are meant to do as we change files as per Eclipse Foundation guidance at https://www.eclipse.org/projects/handbook/#ip-copyright-headers
This is how the other changes made post donation have been handled.
Further advice on the ee4j-pmc list is that the years do not need to be updated.

Reply:
Thank you for explanation.